pager.c

```diff
@@ -286,13 +287,20 @@ static const struct Mapping PagerNewsHelp[] = {
     break;                                                                     \
   }
 
-#define CHECK_READONLY                                                         \
-  if (!Contex2 || Contex2->mailbox->readonly)                                  \
-  {                                                                            \
-    mutt_flushinp();                                                           \
-    mutt_error(_(Mailbox_is_read_only));                                       \
-    break;                                                                     \
-  }
+
+inline static bool assert_mailbox_writable(struct Mailbox *mailbox)
```

Comment:
Do we really need `assert` in the function name?
Would `is_mailbox_writable()` do?